integrationtests/src/test/java/io/kroxylicious/proxy/KroxyConfigBuilder.java

```diff
@@ -80,25 +66,39 @@ public KroxyConfigBuilder withDefaultCluster(String bootstrapServers) {
     }
 
     public KroxyConfigBuilder withMicrometerBinder(String binder) {
-        MicrometerConfig original = currentOrNewMicrometerConfig();
-        micrometer = original.withBinder(binder);
+        MicrometerConfig config = getMicrometerConfigForType("StandardBinders");
+        @SuppressWarnings("unchecked")
+        List<String> binderNames = (List<String>) config.config.computeIfAbsent("binderNames", s -> new ArrayList<String>());
+        binderNames.add(binder);
         return this;
     }
 
-    public KroxyConfigBuilder withMicrometerConfigHook(String className) {
-        MicrometerConfig original = currentOrNewMicrometerConfig();
-        micrometer = original.withConfigHook(className);
+    public KroxyConfigBuilder withMicrometerCommonTag(String key, String value) {
+        MicrometerConfig config = getMicrometerConfigForType("CommonTags");
+        @SuppressWarnings("unchecked")
+        Map<String, String> commonTags = (Map<String, String>) config.config.computeIfAbsent("commonTags", s -> new HashMap<String, String>());
```

Comment:
nit: using `CommonTags` on line 77 and then `commonTags` here seems a bit inconsistent.